omod/src/main/webapp/editIdentifierSource.jsp

```diff
@@ -6,6 +6,21 @@
 <%@ include file="/WEB-INF/template/header.jsp"%>
 <%@ include file="localHeader.jsp"%>
 
+<script type="text/javascript">
+function beforeSubmit() {

```

Comment:
Do you expect this to be named `sanitizeAndSubmit`?

Reply:
@dkayiwa  I have just changed the method call to beforeSubmit to allow flexibility in the future. In case there is a need of more client side controls on the form before submission

Reply:
The original method name was better because it described what exactly the method is doing.
It is not a good practice to sacrifice clarity for a possible future which in most cases turns out not to happen. 😊 
If that future comes and we find that we need to rename/refactor, let it happen then, but not now.

Reply:
@dkayiwa  corrected to original

Reply:
@dkayiwa  I had tested it, and before pushing, I renamed the method to beforeSubmit. That’s how it ended up breaking. I forgot that this wouldn’t rename properly, since the method call in the form tag was being recognized as a string rather than an actual method call in IntelliJ. As a result, the automatic rename didn’t work.